src/alembic_utils/experimental/_collect_instances.py

```diff
@@ -41,7 +41,7 @@ def walk_modules(module: ModuleType) -> Generator[ModuleType, None, None]:
                     # Example: elt.settings
                     module_import_path = str(module_path)[
                         len(str(top_path)) - len(top_module.__name__) :
-                    ].replace("/", ".")[:-3]
+                    ].replace("/", ".").replace("\\", ".")[:-3]
```

Comment:
Could a single `replace` using `os.sep` as the first argument work here instead?